index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html class="box-border">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./input.css">
+    <title>How to play</title>
+</head>
+<body class="flex flex-col m-auto bg-no-repeat bg-cover bg-fixed bg-[url('/img/AdobeStock_239258811.svg')]">
+    <div class="navbar flex justify-between items-center my-2 mx-5 flex-row py-2 px-4 ">
+
+        <div class="logos items-center ml-[18%] xl:ml-[10%] l:ml-[0px]"> 
+            <img class="Experios_logo mr-[26px] h-[30px] items-center inline-block m:self-center m:ml-0" src="img/logo1.svg" alt="Logo">
+            <img class="VirtualEscape_logo h-[34px] mt-1.5 inline-block m:self-center m:ml-0" src="img/logo2.svg" alt="Logo">
```

Comment:
what does m:self-center do ?

Reply:
m is a size of template and I used it for media query and it's defined in tailwind.config.js